### ISSUE TITLE
Certain nodes not available in ComfyUI

### ISSUE THREAD

**Issue by aifuzz59** [ZHO-ZHO-ZHO/ComfyUI-InstantID#10]
I did the install and when I open the workflow (local install) it says that there are missing custon nodes :

Ipadapter_instantidLoader
IDBaseModelLoader_local
IDControlNetLoader
ID_Prompt_Styler
IDGenerationNode

they do not come up on the list of custom missing nodes to install.  How can I install theses nodes?

Thanks

**Comment by kaitmax** [comment]
i have the same issue

**Comment by Hansynily** [comment]
Solved mine. It says missing module named diffusers, so just simply install diffusers

on cmd:
python_embeded\python.exe -m pip install -r ComfyUI\custom_nodes\ComfyUI-InstantID\requirements.txt

**Comment by astragartist** [comment]
I have already installed python_embeded\python.exe -m pip install -r ComfyUI\custom_nodes\ComfyUI-InstantID\requirements.txt but there's still the same problem

**Comment by aifuzz59** [comment]
Yes I installed all the requirements but the certain nodes are stillMissing, googled them and can’t find them anywhere’s Sent from my iPhoneOn Jan 23, 2024, at 2:59 PM, astragartist ***@***.***> wrote:﻿
I have already installed python_embeded\python.exe -m pip install -r ComfyUI\custom_nodes\ComfyUI-InstantID\requirements.txt but there's still the same problem

**Comment by johndpope** [comment]
try this 
https://github.com/ZHO-ZHO-ZHO/ComfyUI-InstantID/pull/17/files

**Comment by lanzer** [comment]
Note that any workflows you put into ComfyUI might require nodes outside of this specific repo you've installed.

No matter what you're installing, be sure to install the ComfyUI Manager first (https://github.com/ltdrdata/ComfyUI-Manager) and then once you drop in a new workflow, open up the manager and select "Install Missing Cusom Nodes".

**Comment by johndpope** [comment]
Thanks @lanzer  - it is not  clear that it can autoresolve.
